index.html

```diff
@@ -3,18 +3,47 @@
 
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="description"
-  content="full description of the app">
-  <meta name="robot" content="index,follow">
-  <link rel="stylesheet" href="styles/reset.css">
-  <link rel="stylesheet" href="styles/debug.css">
-  <title>Document | Description</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
 </head>
 
 <body>
-  <h1>Hello, World!</h1>
+  <main id="main">
+    <h1 id="title">Dr. Norman Borlaug</h1>
+    <figure id="img-div">
+      <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+        alt="the person">
+      <figcaption id="img-caption">increase wheat yields - part of his life-long war on hunger.</figcaption>
+    </figure>
+
+    <div id="tribute-info">
+      <p> Here's a time line of Dr. Borlaug's life:</p>
+      <ul>
+        <li>
+          1914 - Born in Cresco, Iowa
+        </li>
+        <li>
+          1933 - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program
+          known as the "National Youth Administration"
+        </li>
+        <li>
+          1935 - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping
+          starving Americans. "I saw how food changed them", he said. "All of this left scars on me."
+        </li>
+      </ul>
+
+    </div>
+
+    <p>
```

Comment:
- [ ] it would be better to add a class for the `p` tag and be more descriptive. 👍